src/pages/Sign/sections/SignUp.tsx

```diff
@@ -116,21 +117,15 @@ interface Props {
   errorMessage?: string
 }
 
-const validatePersonalNumber = (value: any) => {
-  let errorMessage
-  if (!/^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/i.test(value)) {
-    errorMessage = 'Whops! Är personnummret 10 siffor?'
-  }
-  return errorMessage
-}
-
-const validateEmail = (value: any) => {
-  let errorMessage
-  if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(value)) {
-    errorMessage = 'Whops! Formatet verkar vara fel på angiven email.'
-  }
-  return errorMessage
-}
+const UserSchema = Yup.object().shape({
+  personalNumber: Yup.string()
+    .min(12, 'Personnumret måste vara 12 siffor')
+    .max(12, 'Whops! Är det endast 12 siffor?')
+    .required('Krävs för signering'),
```

Comment:
maybe max 13? if someone adds ie. a dash?
alternatively something like `.matches(/^\d{8}[\s\-]?\d{4}$/, ...)` instead?

Reply:
agree, solved and changed to interval between 6-8 to allow for 10 and 12 digit formats!

Reply:
ah good!

Reply:
nice!

Reply:
just dont forget text keys for validation messages too :D